### ISSUE TITLE
server connection lost when updating an app

### ISSUE THREAD

**Issue by HeinAtCERN** [cznweb/genesis#517]
I clicked on the 'update' button for the Wallabag app. The update was processed properly. The issue only is that when I tried to go somewhere else in the gui, the server connection was lost until the update finished. I guess this is simply because the update is running in the same thread and process as the server itself. 
Maybe for now it would be nice to block the webinterface completely until an update is done. Later, a second thread or process for the update would be cool. But I guess this is quite a bit to implement, as one has to unload/disable the app first, the load it back after the update.

**Comment by peakwinter** [comment]
What version is this for? 0.7 should already have installation and updates done in parallel threads.

**Comment by HeinAtCERN** [comment]
Good point, sorry for not checking. I'm using the vagrant box, which pulls changes on starting, right? But I did ``sudo pacman -Syu`` and the npm installation only after I wrote the issue.
